### ISSUE TITLE
Add LittleNavMap Webserver view

### ISSUE THREAD

**Issue by aduboy** [Christian1984/vfrmap-for-vr#30]
Hello,

Thank you for your excellent tool. It is great for flying in VR. Good job!

Would it be possible in the future to add a layer with the LittleNavMap map view?  (it has a Webserver map that can be viewed in a browser)

Thank you very much!

**Comment by Christian1984** [comment]
Yeah, I started fiddling around with this idea a while ago. There's a separate branch (https://github.com/Christian1984/vfrmap-for-vr/tree/lnm-integration) which is a lot behind `master`, though. I'll eventually revisit it...
